# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpfe/appshell/nsIXULBrowserWindow.idl
# Commit: 2203b9e3146e
# Full Hash: 2203b9e3146ed5567b59718d8904ff77cfc352aa
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-04-08 03:36:50
# Regressor Bug: 1616353
# File Overlap Count: 1
# Description:
#   Bug 1616353 - Part 8: Remove forceInitialBrowserNonRemote, r=Gijs
#   
#   This method is no longer necessary, as frontend will always make the correct
#   decisions for the remoteness of the intial content browser by reading
#   openWindowInfo.
# ==============================================================================

diff -r fbc669bbf02f -r 2203b9e3146e xpfe/appshell/nsIXULBrowserWindow.idl
--- a/xpfe/appshell/nsIXULBrowserWindow.idl	Tue Apr 07 21:39:44 2020 +0000
+++ b/xpfe/appshell/nsIXULBrowserWindow.idl	Tue Apr 07 21:39:47 2020 +0000
@@ -43,13 +43,6 @@
                                 in boolean isAppTab);
 
   /**
-   * Find the initial browser of the window and set its remote attributes.
-   * This can be used to ensure that there is a browser in a new window of the
-   * correct type when it first spawns.
-   */
-  void forceInitialBrowserNonRemote(in mozIDOMWindowProxy openerWindow);
-
-  /**
    * Determines whether a load should continue.
    *
    * @param aDocShell
